vshard/consts.lua

```diff
@@ -47,6 +47,8 @@ return {
     DEFAULT_BUCKET_SEND_TIMEOUT = 10,
     DEFAULT_BUCKET_RECV_TIMEOUT = 10,
 
+    MIGRATION_NOTIFY_SPACE_NAME = '_vshard_migration_notify_',
```

Comment:
Better name it `_PREFIX` since complete name is concatenated lately with engine name

Reply:
Fixed